datadog_checks_dev/datadog_checks/dev/tooling/e2e/local.py

```diff
@@ -121,8 +121,10 @@ def detect_agent_version(self):
 
     def start_agent(self):
         command = get_agent_service_cmd(self.agent_version, self.platform, 'start')
-        return run_command(command, capture=True)
+        shell = True if isinstance(command, string_types) else False
```

Comment:
Why?

Reply:
At some point I saw that you needed shell if the first argument here is a string, but thats indeed not the case, I reverted this.